src/screens/gallery/examples/continuously-animating-line.example.js

```diff
@@ -1,62 +0,0 @@
-/* NOTE
```

Comment:
Yes, the gallery generally needs to be audited. The examples there were placeholders that Paula grabbed from my list of fiddles. They aren't really organized or carefully considered.